sdk-python/src/omnilog/send_to_db.py

```diff
@@ -1,18 +1,48 @@
+import datetime
 import psycopg2
 
+
 def send_to_db(url, log):
+    check_url_type(url)
+    check_log_type(log)
+
     connection, cursor = connect_to_db(url)
 
-    cursor.execute("""
-    INSERT INTO llm_logs (datetime_utc, input_string, output_string, total_tokens)
+    cursor.execute(
+        """
+    INSERT INTO llm_logs (datetime_utc, input_string, output_string, total_tokens) this is a test for extra long string
     VALUES (%s, %s, %s, %s);
-    """, (log["datetime_utc"], log["input"], log["output"], log["total_tokens"]))
+    """,
+        (log["datetime_utc"], log["input"], log["output"], log["total_tokens"]),
+    )
 
     connection.commit()
     cursor.close()
     connection.close()
 
 
+def check_url_type(url):
+    if type(url) != str:
+        raise TypeError("Have you checked your url connection string is correct?")
+
+
+def check_log_type(log):
+    if type(log) != dict:
+        raise TypeError("log must be a dictionary")
```

Comment:
Can we link some docs indicating what the log object should be specifically? Rather than just an object